etc/kayobe/kolla/config/ironic/ironic-conductor.conf

```diff
@@ -0,0 +1,14 @@
+[pxe]
+image_cache_size = 81920
+image_cache_ttl = 10080
```

Comment:
Haven't had to tweak these before. What was the issue?

Reply:
For the very large (20GB) image we push for our main compute cluster, the default Conductor image cache size wasn't large enough, so behaviour we would see would be the Conductor would fail to fully pull the image from Glance and fail a provision. TTL I bumped up to keep the image in cache a bit longer.